source/rust_verify/src/rust_to_vir_func.rs

```diff
@@ -147,19 +218,23 @@ pub(crate) fn check_item_fn<'tcx>(
             );
         }
 
-        let typ = mid_ty_to_vir(ctxt.tcx, is_mut.unwrap_or(input), false);
+        let typ: Typ = mid_ty_to_vir(ctxt.tcx, is_mut.unwrap_or(input), false);
```

Comment:
Why this change?

Reply:
I think  had some trouble with my LSP and was trying to figure out what was causing {unknown} types to appear, this was an artifact of my LSP debugging that I forgot to remove